### ISSUE TITLE
invalid url gives a jq error message

### ISSUE THREAD

**Issue by ramonskie** [frodenas/alertmanager-resource#3]
when you provide a url that is invalid or that doesn't have the username/password when needed
you get a jq error `invalid numeric literal`
so https://github.com/frodenas/alertmanager-resource/blob/master/assets/out#L25-L30
will generate a empty string
that will fail on line https://github.com/frodenas/alertmanager-resource/blob/master/assets/out#L59

a check for the url that you get a 200 would make sense and improve the error message

**Comment by frodenas** [comment]
Agreed, willing to submit a PR to fix this at the [common.sh](https://github.com/frodenas/alertmanager-resource/blob/master/assets/common.sh#L5) script?

**Comment by ramonskie** [comment]
we also found an issue that you can't use the external url of the alertmanager.
so need to figure out how we can do a check for that.
any ideas?
